docs/05-concepts/06-database/03-relations/06-modules.md

```diff
@@ -0,0 +1,59 @@
+# Relations with modules
+
+Serverpod [modules](concepts/modules) usually come with predefined tables and data structures. Sometimes it can be useful to extend them with your data structures by creating a relation to the module tables. Relations to modules come with some restrictions since you do not own the definition of the table, you cannot change the table structure of a module table.
+In practice, this means you are never able to create a many-to-many relation as that would require a modification of the module model definition.
```

Comment:
Isn't it possible to create a many to many relation with a module using a junction table? That could be utilized without modifying the related table?

Reply:
But in this case you don't have a many-to-many with a UserInfo object, really. But you have it with your "shadow" UserInfo table.

Like you are not able to have an object relation on the UserInfo side either.